env/includes/shotgun/all.yml

```diff
@@ -43,3 +43,28 @@ shotgun.publishedfile:
       location: "@common.apps.tk-shotgun-launchfolder.location"
 
   location: "@common.engines.tk-shotgun.location"
+
+shotgun.version:
```

Comment:
I think we should move this whole section to a file called `version.yml` to stay consistent with the rest of the config

Reply:
Agreed -- fixed.